requirements.txt

```diff
@@ -1,4 +1,5 @@
 alembic==1.8.1
+alive-progress==3.1.4
```

Comment:
```suggestion
```

Reply:
Addressed in : [03dc223](https://github.com/MLDSAI/OpenAdapt/pull/318/commits/03dc2237d5dd7fe51ed35db631bd8cde8b908f23)